src/components/ReadMore/index.js

```diff
@@ -49,9 +49,15 @@ const ReadMore = ({text, onPress}) => {
       ) : null}
       {!isFinishSetLayout ? (
         <TouchableOpacity testID="finishLayout" onPress={onPress}>
-          <Text style={styles.text} testID="notFinishLayout" onTextLayout={handleLayoutText}>
-            {text}{' '}
-          </Text>
+          <TextWithEmoji
+            testId="notFinishLayout"
+            text={text}
+            textStyle={{
+              marginBottom: 0,
+              fontSize: normalizeFontSize(14),
+              lineHeight: undefined
+            }}
+          />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The replacement of the `Text` component with `TextWithEmoji` is a good move if emojis need to be supported in the text. The `textStyle` prop allows for custom styling which is a good addition for flexibility. However, setting `lineHeight` to `undefined` could lead to inconsistent rendering across different platforms or screen sizes. If there's a specific reason for this, it would be beneficial to add a comment explaining why.
```diff
-          <Text style={styles.text} testID="notFinishLayout" onTextLayout={handleLayoutText}>
-            {text}{' '}
-          </Text>
+          <TextWithEmoji
+            testId="notFinishLayout"
+            text={text}
+            textStyle={{
+              marginBottom: 0,
+              fontSize: normalizeFontSize(14),
+              lineHeight: undefined
+            }}
+          />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->